.ci/Jenkinsfile

```diff
@@ -318,8 +318,10 @@ def generateFunctionalTestStep(Map args = [:]){
               sh script: """.ci/scripts/install-test-dependencies.sh "${suite}" """, label: "Install test dependencies for ${suite}:${tags}"
             }
           }
-          dir("${BASE_DIR}"){
-            sh script: """.ci/scripts/functional-test.sh "${suite}" "${tags}" "${stackVersion}" "${METRICBEAT_VERSION}" """, label: "Run functional tests for ${suite}:${tags}"
+          filebeat(workdir: "${env.WORKSPACE}"){
```

Comment:
@kuisathaverat I'm gonna set the `output` log file per parallel branch here, to avoid multiple nodes storing logs in the same file name `docker_logs.log`

Reply:
Side feedback: I suggest to make it a default option. Single file merging logs from multiple containers is a bit messy and without grep preprocessing it's hard to analyze.